src/en/darthsdroids/src/eu/kanade/tachiyomi/extension/en/darthsdroids/DarthsDroids.kt

```diff
@@ -0,0 +1,222 @@
+package eu.kanade.tachiyomi.extension.en.darthsdroids
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.model.UpdateStrategy
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+// Dear Darths & Droids creators:
+// I’m sorry if this extension causes too much traffic for your site.
```

Comment:
Add a ratelimit to `client` instead, if the site owner complains it can be tweaked.

Reply:
Good idea, will try.

Reply:
https://github.com/search?q=repo%3Akeiyoushi%2Fextensions-source+client+%3D+network.cloudflareclient+ratelimit&type=code

Reply:
It’s not a CloudFlare site, though. Or is that rate limiter usable anywhere?

Reply:
Not all apps have added the Cloudflare interceptor by default,
Mihon has it enabled: https://github.com/mihonapp/mihon/blob/22e8050fff273a189d2fdfae6e66ec622f04f1dd/core/src/main/java/eu/kanade/tachiyomi/network/NetworkHelper.kt#L47
J2K does not: https://github.com/Jays2Kings/tachiyomiJ2K/blob/8946395f78d22e44471497d3b76de661e9b4166e/app/src/main/java/eu/kanade/tachiyomi/network/NetworkHelper.kt#L54

You should use `network.cloudflareClient` for consistency. At least that's what the norm is, AFAIK.

Reply:
Added to the next commit.